dev/packages/alpha/cisco-0.0.2/manifest.yml

```diff
@@ -0,0 +1,49 @@
+format_version: 1.0.0
+name: cisco
+title: Cisco
+version: 0.0.2
+license: basic
+description: Cisco Integration
+type: integration
+categories:
+  - logs
+release: ga
+removable: true
+requirement:
+  kibana:
+    versions: ">=7.0.0-SNAPSHOT <8.0.0"
+  elasticsearch:
+    versions: ">7.0.1"
+screenshots:
+  - src: /img/kibana-cisco-asa.png
+    title: kibana cisco asa
+    size: 1800x1559
+    type: image/png
+icons:
+  - src: /img/cisco.svg
+    title: cisco
+    size: 216x216
+    type: image/svg+xml
+datasources:
+  - name: cisco
+    title: Cisco logs
+    description: Collect logs from Cisco instances
+    inputs:
+      - type: logs
```

Comment:
As I see here just one input and not multiple, I assume logs (file) is the one we should start with? Long term if we support multiple inputs, we need to have here more then one.